From 1993fdf3850f5d52560b3406362381bcb6e218ef Mon Sep 17 00:00:00 2001
From: Christopher Meng <i@cicku.me>
Date: Sun, 22 Feb 2015 10:29:17 +0800
Subject: [PATCH 2/2] Fix broken quoted string syntax under ocaml 4.02

---
 src/utils/net/terminal.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/utils/net/terminal.ml b/src/utils/net/terminal.ml
index 44a96f4..c9ecde8 100644
--- a/src/utils/net/terminal.ml
+++ b/src/utils/net/terminal.ml
@@ -185,7 +185,7 @@ module Output = struct
   let canal = List.assoc chan !chanmap in
   etat.chan <- Some (chan, canal);
   info (sprintf "connecte au canal '%s'" chan);
-  (* {| canal ALL } *)
+  (* {| canal ALL |} *)
        with Not_found -> (* pas encore de recepteur pour ce canal *)
   try (* on en cree un *)
     let serveur = nsrecord.get_loc chan in
-- 
2.3.0

